includes/class-wc-stripe-account.php

```
@@ -372,6 +372,41 @@ public function delete_previously_configured_webhooks( $exclude_webhook_id = ''
+			return false;
+		}
+
+		// Only decommission when the secret key is being removed or has changed.
+		if ( ! empty( $new_secret_key ) && $new_secret_key === $webhook_data['secret'] ) {
+			return false;
+		}
+
+		try {
+			// Authenticate with the secret key that created the webhook so the deletion
+			// hits the originally connected account.
+			WC_Stripe_API::set_secret_key( $webhook_data['secret'] );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_API' in method 'maybe_decommission_webhook'. (undefined)

(StaticAccess)